include/s3select_oper.h

```diff
@@ -401,10 +401,13 @@ class multi_values
 {
   public:
   std::vector<value*> values;
+  std::vector<std::string> key_path;
```

Comment:
please note the following 
each `value` has json-key-path
`multi_values` is **multiple value** , thus a single key_path may not fit this structure.